modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/configurator/RootProjectConfigurator.java

```diff
@@ -1984,6 +1998,27 @@ private void _configureWorkspaceExtension(
 		}
 	}
 
+	private void _copyTomcatConfiguration(Project project) {
+		WorkspaceExtension workspaceExtension = GradleUtil.getExtension(
+			(ExtensionAware)project.getGradle(), WorkspaceExtension.class);
+
+		File targetAppServerDir = new File(
+			workspaceExtension.getHomeDir(),
+			"tomcat-" + workspaceExtension.getAppServerTomcatVersion());
+
+		File tomcatDir = new File(workspaceExtension.getConfigsDir(), "tomcat");
+
+		try {
+			if (Files.exists(tomcatDir.toPath(), LinkOption.NOFOLLOW_LINKS)) {
+				FileUtils.copyDirectory(tomcatDir, targetAppServerDir);
```

Comment:
@simonjhy I think we can use `project.copy` here instead, right? Is it necessary to pull in `FileUtils`?